### PR TITLE
chore(greenhouse): consolidate admin apps to one component

### PR DESCRIPTION
**Pull request by andypf**
# Summary

This PR consolidates the codebase for Greenhouse by integrating all admin apps under a single source structure within the greenhouse project as components. The consolidated app is now referred to as the Core App and is located under src/components/core-apps. It also introduces React’s lazy loading to load core apps on demand.

# Changes Made

- Moved all package sources from the apps folder to src/components/core-apps/admin.
- Refactored the loading mechanism: core apps are now loaded using React’s lazy loading.
- Merged the latest changes from the main branch.

# Related Issues

- #502 

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

**Review by hodanoori — APPROVED** [review]
I pulled your PR and could run it successfully. Also, I performed quick tests for all apps and they are loaded and working correctly. Since we discussed all my questions in the daily sync meeting, I approve your PR now.

**Comment by ArtieReus** [comment]
I am not able to run this PR locally on Darwing. Encountered errors:

```
Cannot find module @rollup/rollup-darwin-arm64. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try `npm i` again after removing both package-lock.json and node_modules directory.
```
This one I was able to workaround installing manually `npm i @rollup/rollup-darwin-arm64` as proposed also in the [bug report](https://github.com/npm/cli/issues/4828) from npm.

But then the next error popup:

```
[vite:css] [sass] Embedded Dart Sass couldn't find the embedded compiler executable. Please make sure the optional dependency sass-embedded-darwin-arm64 is installed in node_modules.
```

Running following commands didn't help:

```
npm cache clean --force
npm run clean
npm run clean:cache
```

Also not removing the `package-lock.json`

We might have first to fix this before even reviewing the code.

**Comment by andypf** [comment]
> I am not able to run this PR locally on Darwing. Encountered errors:
> 
> ```
> Cannot find module @rollup/rollup-darwin-arm64. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try `npm i` again after removing both package-lock.json and node_modules directory.
> ```
> 
> This one I was able to workaround installing manually `npm i @rollup/rollup-darwin-arm64` as proposed also in the [bug report](https://github.com/npm/cli/issues/4828) from npm.
> 
> But then the next error popped up.:
> 
> ```
> [vite:css] [sass] Embedded Dart Sass couldn't find the embedded compiler executable. Please make sure the optional dependency sass-embedded-darwin-arm64 is installed in node_modules.
> ```
> 
> Running following commands didn't help:
> 
> ```
> npm cache clean --force
> npm run clean
> npm run clean:cache
> ```
> 
> Also not removing the `package-lock.json`
> 
> We might have first to fix this before even reviewing the code.

I will test it on my machine (Mac)

**Review by hodanoori — DISMISSED** [review]
I tested it locally on my WS, every thing works fine. 
lgtm

**Review by barsukov — COMMENTED** [review]
Big PR :) I leave a comment we should make a decission to merge all together but really do it ASAP

File: apps/greenhouse/esbuild/plugin_greenhouse_apps.js [hunk 1]
```diff
@@ -9,11 +9,7 @@ const APPS = [
   path.resolve(APPS_ROOT, "supernova"),
   path.resolve(APPS_ROOT, "heureka"),
   path.resolve(APPS_ROOT, "doop"),
-  path.resolve(APPS_ROOT, "greenhouse/apps/org-admin"),
-  path.resolve(APPS_ROOT, "greenhouse/apps/cluster-admin"),
-  path.resolve(APPS_ROOT, "greenhouse/apps/plugin-admin"),
-  path.resolve(APPS_ROOT, "greenhouse/apps/secret-admin"),
-  path.resolve(APPS_ROOT, "greenhouse/apps/team-admin"),
+  // path.resolve(APPS_ROOT, "greenhouse/core-apps/org-admin"),
```
[the remark this answers is not in the document]
Comment (andypf) [reply apps/greenhouse/esbuild/plugin_greenhouse_apps.js:12]: sure

**Review by barsukov — APPROVED** [review]

**Review by ArtieReus — APPROVED** [review]
lgtm

**Pull request merged**